src/libsync/abstractnetworkjob.h

```diff
@@ -88,6 +88,9 @@ class OWNCLOUDSYNC_EXPORT AbstractNetworkJob : public QObject
      */
     QString errorStringParsingBody(QByteArray *body = 0);
 
+    /** static variable the HTTP timeout (in seconds)  */
```

Comment:
The docs could state that setting it to 0 means using the default value of 5min. would it make sense to initialize it to 300 and then check the env variable in application.cpp? The current way with 0-means-default is a bit convoluted.

Reply:
I know. I did it like this because I wanted to still read from the env variable for owncloudcmd.